### DISCUSSION TITLE
All canonical by i18n goes with trailing slash - how to fix/control that?

### DISCUSSION THREAD

**Discussion by financemarker** [nuxt-modules/i18n#2103] [Q&A] [answered]
### Environment

    "@nuxtjs/i18n": "^7.2.0",
    "nuxt": "^2.15.8",

### Reproduction

Just open Source code of page and check the canonical url

### Describe the bug

For all pages in source code, canonical link has trailing slash at the end. 
How to fix/control that behaviour?

**Comment by kazupon** [comment]
This is question, so we will convert to github discussions

**Reply by kazupon** [reply, marked as answer]
nuxt i18n module has `tralingSlash` option.
You can enable with `trailingSlash: true` on nuxt config.

**Reply by financemarker** [reply]
Yes, but it is related to routes itself, and the problem is with canonical placed by i18n. 
It put trailing slash that I don't need. 
Configuration that I want to achieve - both routes should work example and example/ but canonical should be example
